extensions/reactive-pg-client/deployment/pom.xml

```diff
@@ -15,6 +15,11 @@
 
     <name>Quarkus - Reactive PostgreSQL Client - Deployment</name>
 
+    <properties>
+        <postgres.image>postgres:10.5</postgres.image>
+        <reactive-postgres.url>vertx-reactive:postgresql:///hibernate_orm_test</reactive-postgres.url>
```

Comment:
out of curiosity, why do we need 2 `reactive-postgres.url` properties with different values in the pom?

- `vertx-reactive:postgresql:///hibernate_orm_test`
- `vertx-reactive:postgresql://:5431/hibernate_orm_test`

Reply:
The first one is when Docker is not used: we use the default port of the global instance.

The second one is for Docker as we put it on another port.